pack/tic/def_encounter.json

```diff
@@ -0,0 +1,598 @@
+		"illustrator": "Mark Molnar",
+		"name": "The Devil of the Depths",
+		"pack_code": "def",
+		"position": 166,
+		"quantity": 1,
+		"stage": 2,
+		"text": "As an additional cost for you to move from one [[Ocean]] location to another while not in a vehicle, you must spend 2 actions.\n<b>Forced</b> - When your turn begins, if you are at a fully flooded location, you struggle for air: When your turn ends, if you did not enter an unflooded location, partially flooded location, or a vehicle during your turn, take 5 direct damage.",
+		"type_code": "agenda"
+	},
+	{
+		"back_text": "<b>(→R1)</b>",
+		"back_flavor": "Confident that you've found enough evidence of the Order's activities, you head back to the coast. You'd much rather continue your investigation on dry land.",
+		"back_name": "Secrets Uncovered",
+		"clues": null,
+		"code": "07167",
+		"double_sided": true,
+		"encounter_code": "devil_reef",
+		"encounter_position": 5,
+		"faction_code": "mythos",
+		"flavor": "Somewhere in these accursed reefs, the Esoteric Order of Dagon is hiding something big... but what?",
+		"illustrator": "Alan Dillingham",
+		"name": "Reef of Mysteries",
+		"pack_code": "def",
+		"position": 167,
+		"quantity": 1,
+		"stage": 1,
+		"text": "Keys can only be claimed from [[Y'ha-nthlei]] locations using the [free] ability on those locations.\n<b>Objective</b> - Find and claim as many of the following keys as you can from [[Y'ha-nthlei]] locations (purple, white, and black). If you control all 3 of these keys, advance.",
+		"type_code": "act"
+	},
+	{
+		"back_text": "Churning Waters enters play fully flooded <i>(even while it is unrevealed)</i>.",
+		"back_flavor": "\"And far out at sea, despite a high tide, I glimpsed a long, black line scarcely rising above the water yet carrying a suggestion of odd latent malignancy.\"\n- H. P. Lovecraft, <u>The Shadow over Innsmouth</u>",
+		"clues": 0,
+		"code": "07168",
+		"double_sided": true,
+		"encounter_code": "devil_reef",
+		"encounter_position": 6,
+		"faction_code": "mythos",
+		"illustrator": "Adam Lane",
+		"name": "Churning Waters",
+		"pack_code": "def",
+		"position": 168,
+		"quantity": 1,
+		"shroud": 6,
+		"text": "[action] If you are in the Fishing Vessel: <b>Resign.</b> Each investigator in the Fishing Vessel resigns. Remove Fishing Vessel from the game. Each resigning investigator places each of their purple, black, or white keys on the current act. All other investigators are left to fend for themselves.",
+		"traits": "Ocean.",
+		"type_code": "location"
+	},
+	{
+		"back_flavor": "\"...as I glimpsed it I could not help thinking of all the hideous legends I had heard in the last twenty-four hours - legends which portrayed this ragged rock as a veritable gateway to realms of unfathomed horror and inconceivable abnormality.\"\n- H. P. Lovecraft, <u>The Shadow over Innsmouth</u>",
```

Comment:
Please remove the linebreak where the quotation ends (\n- will create a bullet, so just leave - and no linebreak).